app/controllers/welcome_controller.rb

```diff
@@ -1,4 +1,5 @@
 class WelcomeController < ApplicationController
+  skip_before_action :authenticate_user!
```

Comment:
@ianroberts131 consider skipping authentication in such a way so the developer is always forced to be explicit about which controller actions don't need authentication:

```diff
-  skip_before_action :authenticate_user!
+  skip_before_action :authenticate_user!, only: :home
```

This is a more secure default than skipping authentication for all actions in `WelcomeController` (admittedly it only has one action currently so it makes little difference at this moment). 

In the future, if a new `WelcomeController#top_secret` action was added then we want it to require authentication by default in case we're forgetful about re-enabling authentication.